tests/unit/views/PasswordCheck.spec.js

```diff
@@ -1,72 +1,157 @@
+    if (passwordStrengthIndicatorWrapper && passwordStrengthIndicatorWrapper.exists()) {
+      passwordStrengthIndicatorWrapper.destroy();
+    }
   });
 
   it('should render', () => {
     expect(wrapper.exists()).toBeTruthy();
   });
 
   it('should contain password field', () => {
-    expect(wrapper.get('[data-test="password-field"]').attributes('type')).toEqual('password');
+    expect(inputWrapper.get('[data-test="password-field"]').attributes('type')).toEqual('password');
   });
 
   describe('rules', () => {
     describe('when password has one letter', () => {
       beforeEach(async () => {
-        await setPasswordTo('a');
+        store.commit('setPassword', 'a');
       });
 
-      it.todo('should highlight one letter is satisfied');
-      it.todo('should not highlight number satisfied');
+      it('should highlight one letter is satisfied', () => {
+        expect(rulesWrapper.get('[data-test-rule="OneLetter"]').classes('rules-list__item--checked')).toBe(true);
+      });
+      it('should not highlight number satisfied', () => {
+        expect(rulesWrapper.get('[data-test-rule="OneNumber"]').classes('rules-list__item--checked')).toBe(false);
+      });
     });
 
     describe('when password has lower an upper letters', () => {
       beforeEach(async () => {
-        await setPasswordTo('aB');
+        store.commit('setPassword', 'aB');
       });
 
-      it.todo('should highlight one letter is satisfied');
-      it.todo('should highlight upper and lower letter is satisfied');
-      it.todo('should not highlight number satisfied');
+      it('should highlight one letter is satisfied', () => {
+        expect(rulesWrapper.get('[data-test-rule="OneLetter"]').classes('rules-list__item--checked')).toBe(true);
+      });
+      it('should highlight upper and lower letter is satisfied', () => {
+        expect(rulesWrapper.get('[data-test-rule="UpperAndLower"]').classes('rules-list__item--checked')).toBe(true);
+      });
+      it('should not highlight number satisfied', () => {
+        expect(rulesWrapper.get('[data-test-rule="OneNumber"]').classes('rules-list__item--checked')).toBe(false);
+      });
     });
   });
 
 
   describe('strength validation', () => {
     it('should be rendered', () => {
-      expect(wrapper.find('[data-test="password-strength"]').exists()).toBeTruthy();
+      expect(passwordStrengthIndicatorWrapper.find('[data-test="password-strength"]').exists()).toBeTruthy();
     });
 
     describe('when less than 5 rules satisfied', () => {
       const examples = ['', 'aaa', 'aB$4'];
 
       examples.forEach((password) => {
         it(`should show that password "${password}" is weak`, async () => {
-          await setPasswordTo(password);
+          await store.commit('setPassword', password);
 
-          expect(wrapper.get('[data-test="password-strength"]').text()).toEqual('Weak');
+          expect(passwordStrengthIndicatorWrapper.get('[data-test="password-strength"]').text()).toEqual('Weak');
```

Comment:
```suggestion
          expect(passwordStrengthIndicatorWrapper.get('[data-test="password-strength"]').text()).toBe('Weak');
```
You can use `toBe()` with primitives and `toEqual()` with complex structures.